gamestonk_terminal/portfolio_optimization/README.md

```diff
@@ -1,169 +1,192 @@
 
-##Sample Usage
-In this example, we generate weights for a list of 6 stocks using the eff_ret command.  This optimization looks to maximize returns 
-at a given risk level.  We start by adding the stocks we want to analyze:
+Maximises the quadratic utility, given some risk aversion.
+
+* -p : Amount of time to retrieve data from yfinance. Default: 3mo.
+* -v : If provided, this represents an actual allocation amount for the portfolio.  Defaults to 1, which just returns the weights.
+* --pie : Display a pie chart for weights
+* -r : Risk aversion parameter
+* -n : Whether the portfolio should be market neutral (weights sum to zero). Defaults: False.
+
+<img width="1004" alt="maxquadutil" src="https://user-images.githubusercontent.com/25267873/114958288-9640b380-9e5a-11eb-8b4e-ac61b11db638.png">
+
+
+### effret <a name="effret"></a>
+
 ````
-select aapl,amzn,msft,f,gm,ge
+usage: effret [-p {1d,5d,1mo,3mo,6mo,1y,2y,5y,10y,ytd,max}] [-v VALUE] [--pie] [-t TARGET_RETURN] [-n]
 ````
-Which shows:
+
+Calculate the 'Markowitz portfolio', minimising volatility for a given target return.
+
+* -p : Amount of time to retrieve data from yfinance. Default: 3mo.
+* -v : If provided, this represents an actual allocation amount for the portfolio.  Defaults to 1, which just returns the weights.
+* --pie : Display a pie chart for weights
+* -t : The desired return of the resulting portfolio
+* -n : Whether the portfolio should be market neutral (weights sum to zero). Defaults: False.
+
+<img width="990" alt="effret" src="https://user-images.githubusercontent.com/25267873/114958286-9640b380-9e5a-11eb-986a-0ddf359121ae.png">
+
+
+### effrisk <a name="effrisk"></a>
+
 ````
-Current Tickers: GE, GM, AMZN, AAPL, F, MSFT
+usage: effrisk [-p {1d,5d,1mo,3mo,6mo,1y,2y,5y,10y,ytd,max}] [-v VALUE] [--pie] [-t TARGET_VOLATILITY] [-n]
 ````
-To perform the optimization, we will set a target return of 25% (.25).  Given how stocks performed during COVID,
-there is a lot of volatility, so many optimizations may not get low volatility.  The module also returns annualized volatility,
-so the number is your portfolio volatility * `sqrt(252`.  This optimization will be (including a pie chart!).  Note we could also supply a different
-time period, which changes the expected returns and historical volatility, which changes the optimization.  We could also specify a dollar
-amount that you wish to allocate using the `-v` flag.
+
+Maximise return for a target risk. The resulting portfolio will have a volatility less than the target (but not guaranteed to be equal).
+
+* -p : Amount of time to retrieve data from yfinance. Default: 3mo.
+* -v : If provided, this represents an actual allocation amount for the portfolio.  Defaults to 1, which just returns the weights.
+* --pie : Display a pie chart for weights
+* -t : The desired maximum volatility of the resultingportfolio
+* -n : Whether the portfolio should be market neutral (weights sum to zero). Defaults: False.
```

Comment:
As I mentioned above, curious if this breaks the pie plots when market neutral

Reply:
I'll do some more testing around this. Thanks for pointing it out!

If it does I'll just remove that flag